wiki/Tournaments/ATS/2018/en.md

```diff
@@ -279,132 +286,152 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
   - [t+pazolite - Pumpin' Junkies (Nwolf) \[Nwolf's Lite Oni\]](/beatmapsets/364288/#taiko/991619) 
   - [Atoguru - Privilege (Alace) \[Pangko\]](/beatmapsets/43861/#taiko/138598)
 - Tiebreaker
-  - [Demetori - Desire Drive ~ Desire Dream (OnosakiHito) \[Ono's Taiko Oni\]](/beatmapsets/42234#taiko/135188)
+  - **[Demetori - Desire Drive ~ Desire Dream (OnosakiHito) \[Ono's Taiko Oni\]](/beatmapsets/42234#taiko/135188)**
 
 ------------------------------------------------------------------------
 
 ## Match Results
 
+### Grand Finals
+
+| Group | Sunday, 2018-07-08 |  |  |  |  | 
+| :-: | --: | :-: | :-- | :-- | :-: |
+| UB | nyanmi-1828 ![][flag_JP] | 2 | **7** | ![][flag_TW] **Smallwu** | [#1](/community/matches/43991727) |
+| LB | **Smallwu** ![][flag_TW] | **7** | 0 | ![][flag_JP] nyanmi-1828 | -Win by Default- |
```

Comment:
Dont use caps on Win by Default, better to use lowercase => `-win by default-`
Repeat on every instance.